ini/stm32f1.ini

```diff
@@ -233,6 +233,24 @@ build_unflags               = ${stm32_variant.build_unflags}
 debug_tool                  = jlink
 upload_protocol             = jlink
 
+#
+# ZNP Robin Nano V1.2
+#
+[env:znp_robin_nano35]
```

Comment:
also no need for full copy, just set 
`extends = mks_robin_nano35
board_build.encrypt_mks = elegoo.bin`
also can be `#define FIRMWARE_BIN elegoo.bin` in example config

Reply:
*facepalm* idk why I didn't think of that.. lol